.github/workflows/changelog-check.yml

```diff
@@ -0,0 +1,18 @@
+name: Check Changelog
+
+on:
+  pull_request:
+    types: [opened, synchronize, labeled, unlabeled]
+  merge_group:
+    types: [checks_requested]
```

Comment:
Do we need this? We don't use the merge queue in this repo. I also don't know if `github.event.pull_request` is available for this event.

Reply:
No we don't forget to remove it